Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM centos:6
+
+MAINTAINER IoT team
+
+RUN yum update -y && yum install -y wget
+RUN wget http://dl.fedoraproject.org/pub/epel/6/x86_64/epel-release-6-8.noarch.rpm
+RUN yum localinstall -y --nogpgcheck epel-release-6-8.noarch.rpm
+RUN yum install -y rpm-build git java-1.7.0-openjdk java-1.7.0-openjdk-devel
+ENV JAVA_HOME=/usr/lib/jvm/java-1.7.0-openjdk.x86_64
+
+# Install Maven
+RUN wget -c http://ftp.cixug.es/apache/maven/maven-3/3.2.5/binaries/apache-maven-3.2.5-bin.zip
+RUN unzip -oq apache-maven-3.2.5-bin.zip
+RUN cp -rf apache-maven-3.2.5 /opt/maven
+RUN ln -fs /opt/maven/bin/mvn /usr/bin/mvn
+
+## Install MySQL (client at least?)
+RUN yum -y install mysql
+
+RUN mkdir /github && mkdir /github/telefonicaid
+
+WORKDIR /github/telefonicaid
+RUN git clone https://github.com/telefonicaid/fiware-keypass 
```

Comment:
This isn't needed. Use a COPY instruction to add the local Github code to /opt/keypass